src/components/application_manager/include/application_manager/resumption/resumption_data_processor.h

```diff
@@ -298,6 +298,8 @@ class ResumptionDataProcessor : public app_mngr::event_engine::EventObserver {
   void CheckVehicleDataResponse(const smart_objects::SmartObject& request,
                                 const smart_objects::SmartObject& response,
                                 ApplicationResumptionStatus& status);
+
+  bool HasNoHMIRequestsSent(const int32_t app_id);
```

Comment:
@LitvinenkoIra Please rename to something more clear and meaningful

Reply:
@LitvinenkoIra description missed